src/pages/en/concepts/why-astro.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Why Astro?
+description: DESCRIPTION
+i18nReady: true
+---
+
+Astro is an **all-in-one** **web framework** for building **fast,** **content-focused websites.**
+
+Why choose Astro over another web framework? Here are five core design principles to help explain why we built Astro, the problems that it exists to solve, and why Astro may be the best choice for your project or team.
+
+#### Astro is...
+
+1. [Content-focused](#) -- Astro is designed for building content-rich websites.
+2. [Server-first](#) -- Websites work better when they do more work on the server.
+3. [Fast by default](#) -- It should be impossible to build a slow website in Astro.
+4. [Easy to use](#) -- You don't need to be a JavaScript expert to build with Astro.
+5. [Fully-featured, but flexible](#) -- Extend Astro with over 100+ integrations.
```

Comment:
```suggestion
5. [Fully featured, but flexible](#) -- Extend Astro with over 100+ integrations.
```

Reply:
I'm just googling this phrase, but I think "Fully-featured" is correct.

Reply:
Oops, I missed reply to this one (because I thought it occured elsewhere, and that I replied there)

Sorry, Yan: "Fully-featured" is correct!  (But, don't get me started on "full-featured" vs "fully-featured" ...)  😄